### ISSUE TITLE
Ignore does not work

### ISSUE THREAD

**Issue by davesag** [rasshofer/yaml-lint#19]
My `.yaml-lint.json` is

```
{
  "schema": "CORE_SCHEMA",
  "ignore": "node_modules/**/*.yml"
}
```

But running `yamllint **/*.yml` gives me

```
✖ YAML Lint failed for /src/myproject/node_modules/mute-stream/.travis.yml
✖ duplicated mapping key at line 2, column 1:
    language: node_js
    ^
```

Likewise running

```
yamllint **/*.yml --ignore=node_modules/**/*.yml
```

gives the same error.

How do I get this to ignore everything in `node_modules`?

**Comment by rasshofer** [comment]
Ooops, thanks a lot for the hint! 🙈 This is fixed with v1.2.4.

**Comment by davesag** [comment]
Great that fixed it. Cheers
